PrebidMobile/Logging/Log.swift

```diff
@@ -29,35 +29,46 @@ public class Log: NSObject {
         return formatter
     }
     
+    public static func setCustomLogger(customLogger: PrebidLogger?) {
+        logger = customLogger
+    }
+    
     public static var logLevel: LogLevel = .debug
     public static var logToFile = false
 
     public static func error(_ object: Any, filename: String = #file, line: Int = #line, function: String = #function) {
         log(object, logLevel: .error, filename: filename, line: line, function: function)
+        logger?.error(object, filename: filename, line: line, function: function)
     }
 
     public static func info(_ object: Any, filename: String = #file, line: Int = #line, function: String = #function) {
         log(object, logLevel: .info, filename: filename, line: line, function: function)
```

Comment:
Now, you should remove system fogging from the Log class. Otherwise, the same messages will be logged twice.